App/App.module.css

```diff
@@ -0,0 +1,4 @@
+.app-page{

```

Comment:
Looks great, awesome work! Only change requested is to move this css file under the App folder in the frontend folder. And delete the extra App Folder so that this PR only has the Frontend/app folder.